projects/observability/src/pages/apis/service-detail/instrumentation/service-instrumentation.component.ts

```diff
@@ -1,54 +1,40 @@
-import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { map } from 'rxjs/operators';
 
 import { BreadcrumbsService } from '@hypertrace/components';
-import { BehaviorSubject, Observable } from 'rxjs';
-import { map } from 'rxjs/operators';
 import { ServiceInstrumentationService } from './service-instrumentation.service';
-import { OrgScoreResponse, ServiceScoreResponse } from './service-instrumentation.types';
+import { ServiceScoreResponse } from './service-instrumentation.types';
 
 @Component({
   styleUrls: ['./service-instrumentation.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   providers: [ServiceInstrumentationService],
   template: `
     <main class="service-instrumentation" *ngIf="this.serviceScoreSubject | async">
-      <section class="overview">
-        <ht-service-instrumentation-total-score
-          [serviceScore]="(this.serviceScoreSubject | async)?.aggregatedWeightedScore"
-        ></ht-service-instrumentation-total-score>
-
-        <ht-service-instrumentation-org-score
-          [orgScore]="(this.orgScoreResponse$ | async)?.aggregatedWeightedScore"
-        ></ht-service-instrumentation-org-score>
-      </section>
-
-      <section class="checks-container">
-        <ht-service-instrumentation-category-card
-          *ngFor="let categoryScore of (serviceScoreSubject | async)?.qoiTypeScores"
-        ></ht-service-instrumentation-category-card>
-      </section>
+      <router-outlet></router-outlet>
     </main>
   `
 })
-export class ServiceInstrumentationComponent {
+export class ServiceInstrumentationComponent implements OnInit {
   public serviceScoreSubject: BehaviorSubject<ServiceScoreResponse | undefined> = new BehaviorSubject<
     ServiceScoreResponse | undefined
   >(undefined);
 
-  public orgScoreResponse$: Observable<OrgScoreResponse>;
-
   public constructor(
     private readonly breadcrumbsService: BreadcrumbsService,
     private readonly serviceInstrumentationService: ServiceInstrumentationService
   ) {
+    this.serviceScoreSubject = this.serviceInstrumentationService.serviceScoreSubject;
+  }
+
+  public ngOnInit(): void {
     this.breadcrumbsService.breadcrumbs$
       .pipe(map(breadcrumbs => (breadcrumbs.length > 0 ? breadcrumbs[breadcrumbs.length - 1]?.label : undefined)))
       .subscribe(serviceName => {
         this.serviceInstrumentationService
           .getServiceScore(serviceName!)
-          .subscribe(serviceScore => this.serviceScoreSubject.next(serviceScore));
+          .subscribe(serviceScore => this.serviceInstrumentationService.serviceScoreSubject.next(serviceScore));
```

Comment:
You can do `this.serviceScoreSubject.next` given we are already saving it in a variable. Or remove the variable if not needed.

Reply:
The idea is to update the subject in the common service, not the component. The subject in the component is just a copy of the service one to keep the code shorter, but it's not needed.

Reply:
✅ Replaced variable with a method `getServiceScore()` that returns the subject from the common service.

Reply:
Doing this started failing the Security Check CI for some reason. Can you please check what could be wrong?
The last commit has the change: https://github.com/razorpay/hypertrace-ui/pull/84/commits/e7b2841d6c9790d8e135c66e4a29b2d9a2fbf4b1